gitleaks_baselines/gitleaks-apiv2.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "Description": "",
+    "StartLine": 2585,
+    "EndLine": 2585,
+    "StartColumn": 0,
+    "EndColumn": 0,
+    "Match": "S/Ru0Uau47Ap00munEyJzuDR6zcGfeTH46PXb4AzQ84XGs41Pi7xbiJghr5CFzl4Dr1MAv+sF8UP7dYc",
```

Comment:
Do not seem to be any secret as such. False alarm apparently.

Reply:
https://github.com/open-edge-platform/infra-core/blob/f5f86a742e454598a9247d1993123f79cf7f72e4/apiv2/pkg/api/v2/edge-infrastructure-manager-openapi-server.gen.go#L2585-L2586

Reply:
Try adding an entry to https://github.com/open-edge-platform/infra-core/blob/main/.gitleaksignore. though it would be a moving target